qs-services/qs-panache-service/pom.xml

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>hu.icellmobilsoft.quarkus.sampler</groupId>
+        <artifactId>qs-services</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>qs-panache-service</artifactId>
+
+    <properties>
+        <jvm.args>--add-opens=java.base/java.net=ALL-UNNAMED</jvm.args>
+        <version.io.agroal>2.3</version.io.agroal>
+        <quarkus-micrometer-registry-otlp.version>3.2.4</quarkus-micrometer-registry-otlp.version>
+    </properties>
+
+    <dependencies>
+        <!-- Project dependecies-->
+        <dependency>
+            <groupId>hu.icellmobilsoft.quarkus.sampler</groupId>
+            <artifactId>qs-common-jpa</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>hu.icellmobilsoft.quarkus.sampler</groupId>
+            <artifactId>qs-common-rest</artifactId>
+        </dependency>
+
+        <!-- Quarkus dependencies Telemetry -->
+        <dependency>
+            <groupId>io.opentelemetry.instrumentation</groupId>
+            <artifactId>opentelemetry-jdbc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-extension-trace-propagators</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-opentelemetry</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkiverse.micrometer.registry</groupId>
+            <artifactId>quarkus-micrometer-registry-otlp</artifactId>
+            <version>${quarkus-micrometer-registry-otlp.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-opentelemetry-exporter-otlp</artifactId>
+            <version>3.9.2</version>
+        </dependency>
+<!-- dash not showing data due to missing instance name  -->
+<!--<dependency>-->
```

Comment:
ez végső soron kell vagy nem?

Reply:
ez bugos sztem még, ezért lett kiemelve külön docker compose-ba, és sztem azt is fogjuk használni a jövőben, könnyebb tesztelni és módosítani a dasheket